Index: ../startjava-master/rules.md
===================================================================
diff --git a/../startjava-master/rules.md b/../startjava-master/rules.md
deleted file mode 100644
--- a/../startjava-master/rules.md	
+++ /dev/null	
@@ -1,33 +0,0 @@
-## Правила на курсе StartJava при проверке ДЗ наставником
-
-1. Все друг с другом общаются на Ты
-1. Доступ к видеолекциям предоставляется только к текущему уроку. Последующие уроки открываются по мере завершения текущего (сделано с целью профилактики списывания)
-1. Проверка ДЗ действует 10 недель с момента оплаты с возможностью продления
-1. Взаимодействие с наставником проходит по следующей схеме:
-    - вы смотрите видеолекции (ссылки на них находятся рядом с иконками логотипа ютуба), изучаете дополнительный материал, выполняете ДЗ и **сдаете их по одному (а не все сразу!)**
-    - по мере готовности сообщаете наставнику, что его нужно проверить
-    - он проверяет и пишет, что требуется исправить
-    - вы исправляете, он снова проверяет. И так повторяется до тех пор, пока ваше решение не будет соответствовать условию ДЗ и представлению наставника о том, как его нужно выполнить
-    - если на вашем последнем сообщении наставник ставит 📌, то он его прочитал. Если стоит 🧐, то в данный момент ваше ДЗ на проверке. Если стоит ✔️- ДЗ проверено
-    - если все верно, то наставник напишет, чтобы вы выполняли следующее ДЗ
-1. Ссылку на свой репозиторий наставнику достаточно скинуть один раз
-1. Комментарии к вашему коду будут даваться наставником в краткой форме. Если вам требуется пояснение по какому-то пункту, то задавайте ему дополнительные вопросы для получения более развернутого ответа
-1. Переходить к следующему уроку можно после того, как вам об этом сообщит наставник. Не делайте это самостоятельно
-1. Для минимизации "простоев" допускается выполнять последующие ДЗ до того, как наставник вам об этом напишет
-1. Максимальное время проверки ДЗ до суток, но как правило раньше
-1. Если наставник не проверил ДЗ за указанный срок, то напомните ему о себе (он вас мог случайно пропустить)
-1. Если у вас закончился срок проверок ДЗ, то его можно продлить еще на месяц в личном кабинете (ссылка на него есть в любом нашем письме) [на сайте](https://javaops.ru/)
-1. Доступ к материалам курса у вас остается навсегда
-1. Пришлите наставнику свою гугл-почту для доступа ко второму и последующим урокам
-
-#### Общие рекомендации
-- Учитесь грамотно формулировать свой вопрос: "у меня не работает" может иметь тысячи причин. Пишите больше подробностей, что вы сделали, что конкретно не работает, какие появляются ошибки. Присылайте в чат скрины с этими ошибками
-- Учитесь гуглить с первых уроков - это важный навык, который нужно качать
-- Перед тем как задать вопрос наставнику сначала погуглите; если в течении 2-3 часов вы не найдете ответ, то опишите подробно свою проблему наставнику
-
-#### Расписание проверок ДЗ
-
-[Maksim Mentor](https://startjava-w.slack.com/team/U04V1C4SLBC)
-- выходные дни вторник и четверг
-- время проверки с 11:00 по 13:00 и с 17:30 по 19:30 (но не позднее 21:00 по мск. времени)
-- проверка в выходные дни не гарантируется, но возможна
Index: ../startjava-master/lesson 2.md
===================================================================
diff --git a/../startjava-master/lesson 2.md b/../startjava-master/lesson 2.md
deleted file mode 100644
--- a/../startjava-master/lesson 2.md	
+++ /dev/null	
@@ -1,117 +0,0 @@
-# Занятие второе
-
-## Разбор домашнего задания
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Калькулятор](https://drive.google.com/file/d/1VXg-RlS-DprXOjbH8s5FieOUga_PCzBZ/view?usp=sharing)
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Угадай число](https://drive.google.com/file/d/1XFSXQuUchrwD7aNsUQKjxvRtKNy1_wkx/view?usp=sharing)
-
-**Эталонный код ДЗ 1-го урока:**
-- [VariableNamesTheme](https://drive.google.com/file/d/17Fl3qSSS5l7-CdBiJAraaL5PiJAyj3gv/view?usp=sharing)
-- [VariablesTheme](https://drive.google.com/file/d/1AQYnQs0G2hFx3tM8BN8v75H46I4GPNUD/view?usp=drive_link)
-- [IfElseStatementTheme](https://drive.google.com/file/d/18I_TlL8f7qW45M2t_90re8JtQBn3mHDQ/view?usp=sharing)
-- [CyclesTheme](https://drive.google.com/file/d/1oUH7Y-GGxVrbG7trflIPiZESxJZDyehq/view?usp=drive_link)
-- [RpsGame](https://drive.google.com/file/d/14LZkJu01U1UCZWx4h2J1jdV5C5BOK9u0/view?usp=sharing)
-
-## Темы занятия
-1. [Что такое ООП](#1)
-1. [Что такое класс](#2)
-1. [Что такое объект](#3)
-1. [Инкапсуляция: геттеры/сеттеры](#4)
-1. [Ответы на вопросы](#5)
-1. [Итоговые домашние задания](#6)
-1. [Опросники](#7)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 1. <a name="1">[Что такое ООП](https://drive.google.com/file/d/1Jsfz5Ygb5Ct1in7MTcNV3wSsx_30D5h3/view?usp=sharing)</a>
-
->ООП — это одна из парадигм (методологий, способов оформления кода) программирования, основанная на принципах, предполагающих написание программного кода в виде классов и их объектов, взаимодействующих друг с другом
-
-**Материалы:**
-- ООП с примерами. Часть [1](https://habr.com/post/87119/), [2](https://habr.com/post/87205/)
-- [Зачем нам ООП и что это такое](https://habr.com/post/148015/)
-- [Объектно-ориентированное программирование](http://info.javarush.ru/translation/2016/01/28/Объектно-ориентированное-программирование-перевод-статьи-.html)
-- [Алан Кэй, создатель ООП, про разработку, Лисп и ООП](https://habr.com/company/hexlet/blog/303754/)
-- [Simula — 50 лет ООП](https://habr.com/post/345944/)
-
-![oop](https://user-images.githubusercontent.com/29703461/39483340-b3a50496-4d7a-11e8-8e02-42a8d63de02d.jpg)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 2. <a name="2">[Что такое класс](https://drive.google.com/file/d/1AjEL8wuTK6BbiMWB2hhi_Cx6_m7NMrD1/view?usp=sharing)</a>
-
->Класс — это ссылочный тип данных, являющийся прототипом (blueprint, шаблоном) объекта, описывающий его характеристики и реализующий его поведение
-
-**Материалы:**
-- [Классы в Java](https://javarush.ru/groups/posts/1949-znakomstvo-s-klassami-napisanie-sobstvennihkh-klassov-konstruktorih)
-- [Classes](https://docs.oracle.com/javase/tutorial/java/javaOO/classes.html)
-- [What Is a Class?](https://docs.oracle.com/javase/tutorial/java/concepts/class.html)
-- [Тип данных String в Java](https://habr.com/post/260767)
-
-![class](https://user-images.githubusercontent.com/29703461/113477118-843d3900-9488-11eb-8409-728af529574d.png)
-
-**Домашнее задание:**
-- Создайте класс **Person** (без метода `main`)
-- Напишите в нем поля (атрибуты), описывающие характеристики человека: `пол, имя, рост, вес, возраст`
-- Проинициализируйте их какими-то значениями
-- Создайте в классе методы, описывающие поведение (что он может делать) человека: `идти, сидеть, бежать, говорить, учить Java`
-- В каждом методе используйте `sout` для вывода соответствующего названию метода сообщения
-- Скомпилируйте класс, чтобы проверить его на ошибки (не пытайтесь его запустить, т. к. в нем нет метода `main`)
-- **Для форматирования кода этого и последующих ДЗ используйте правила из [статьи](https://topjava.ru/blog/pravila-formatirovaniya-koda-v-java2)**
-- **Покажите ДЗ наставнику**
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 3. <a name="3">[Что такое объект](https://drive.google.com/file/d/1hJCGeVRy-4mTTkViK219JzfLKbwLze66/view?usp=sharing)</a>
-
->Объект — это экземпляр какого-либо класса, обладающий характеристиками в виде полей (fields) и поведением (функционалом) в виде методов (methods)
-
-**Материалы:**
-- [What Is an Object?](https://docs.oracle.com/javase/tutorial/java/concepts/object.html)
-- [Objects](https://docs.oracle.com/javase/tutorial/java/javaOO/objects.html)
-
-![object](https://user-images.githubusercontent.com/29703461/39529416-4e58a1e6-4e2f-11e8-9a37-029871ea096a.png)
-
-**Домашнее задание:**
-- Создайте класс **Wolf**
-   - объявите в нем поля: `пол, кличка, вес, возраст, окрас` (не присваивайте им значения по умолчанию)
-   - реализуйте методы: `идти, сидеть, бежать, выть, охотиться`
-   - в каждом методе выводите сообщение вида: `"Воет", "Сидит"` и т. д.
-- Создайте класс **WolfTest** с методом **main**
-  - в нем создайте объект типа **Wolf**
-  - в этом же классе присвойте полям **Wolf** какие-то значения
-  - получите эти значения из полей и отобразите в консоли
-  - вызовите методы объекта
-- **Покажите ДЗ наставнику**
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 4. <a name="4">[Инкапсуляция: геттеры/сеттеры](https://drive.google.com/file/d/1GWI8rJS6Xwbhz512R4ohdN3b0UKYnnm1/view?usp=sharing)</a>
-
-**Материалы:**
-- [Что такое инкапсуляция](https://youtu.be/nyFQvgrkoXY) (youtube)
-- [Описание инкапсуляции](https://github.com/ichimax/Java-Interview-Questions/blob/master/Questions/1.%20OOP.md#Что-такое-инкапсуляция)
-- [Инкапсуляция поля](https://refactoring.guru/ru/encapsulate-field) (через VPN)
-- [Инкапсуляция: геттеры и сеттеры](https://youtu.be/9Uo8SYk7lbk) (youtube)
-- [Геттеры и сеттеры в Java](https://topjava.ru/blog/gettery-i-settery-v-java)
-- [Геттеры и сеттеры](https://javarush.ru/groups/posts/1928-getterih-i-setterih)
-
-## 5. <a name="5"> Ответы на вопросы</a>
-> в чем разница между `классом` и `объектом`? Или это одно и тоже?
-
-Класс и объект — это разные понятия. Класс — это «чертеж», на основе которого создаются объекты. Благодаря этому «чертежу» JVM знает, какой объект ей надо создать
-
-**Домашнее задание:**
-- Модифицируйте класс **Wolf**
-   - у всех полей класса пропишите модификатор `private`
-   - а у его методов — `public`
-   - для доступа к полям создайте геттеры и сеттеры
-   - в сеттере поля `age` реализуйте проверку: если возраст волка > 8 лет, то выведите сообщение `"Некорректный возраст"`. В противном случае — сохраните значение в поле
-- Модифицируйте класс **WolfTest**  
-  - с помощью сеттеров присвойте полям экземпляра класса **Wolf** какие-то значения
-  - считайте эти значения из полей с помощью геттеров и отобразите в консоли
-- **Покажите ДЗ наставнику**
-
-## 6. <a name="6">Итоговые домашние задания</a>
-- **[Итоговые домашние задания](https://docs.google.com/document/d/1VJxMqjhRilLIqguoGvb-4T9v1jGxWQwa_KgMWEG-fmE/edit?usp=sharing)**
-
-<details>
-<summary>Так должна выглядеть структура ваших файлов и папок</summary>
- 
-![tree](https://github.com/ichimax/startjava/assets/29703461/4d7bf236-eedd-465f-a687-5193f6fc76fe)
-</details>
- 
- ## 7. <a name="7">Опросники</a>
-- [Итоговый тест](https://forms.gle/F8k6FHKu85dbkGth6) позволит вам проверить, насколько хорошо вы освоили пройденные темы
-- В этом небольшом [опроснике](https://forms.gle/YDyNWaaf7SssGEtAA) вы сможете поделиться своими впечатлениями (анонимно) по поводу второго урока: что (не)понравилось, как его можно улучшить и тд. Ваши ответы помогут нам доработать урок, сделав его еще лучше!
Index: HW_2/Wolf.java
===================================================================
diff --git a/HW_2/Wolf.java b/HW_2/Wolf.java
deleted file mode 100644
--- a/HW_2/Wolf.java	
+++ /dev/null	
@@ -1,33 +0,0 @@
-public class Wolf{
-
-	private String gender = "М";
-	private String name = "Мухтар";
-	private int weigh = 20;
-	private int age = 5;
-	private String color = "Черный";
-
-	public void go(int step){
-		System.out.println("Собака сделала " + step + " шагов");
-	}
-
-	public void set_doun(){
-		System.out.println("Собака сидит");
-	}
-
-	public void sprint(int step){
-		System.out.println("Собака пробежала " + step + " шагов");
-	}
-
-	public void setWeigt(int weigh){
-	this.weigh = weigh;
-	}
-
-	public String getName(){
-		return name;
-	}
-
-	public int getWeigt(){
-		return weigh;
-	}
-
-}
\ No newline at end of file
Index: ../startjava-master/lesson 5.md
===================================================================
diff --git a/../startjava-master/lesson 5.md b/../startjava-master/lesson 5.md
deleted file mode 100644
--- a/../startjava-master/lesson 5.md	
+++ /dev/null	
@@ -1,66 +0,0 @@
-# Занятие пятое
-
-Пора уже смотреть в сторону следующего курса — `BaseJava`. Начинайте потихоньку пробовать выполнить вступительное ДЗ, которое можно получить в личном кабинете на [javaops.ru](https://javaops.ru/): влкадка `Новые`(или `В процессе`) → `Открытое занятие`
-
-**Эталонный код ДЗ 4-го урока:**
-- [ArraysTheme](https://drive.google.com/file/d/1eGOIp7FJ2e-hCxyzLofI0XvDfIj-f7xP/view?usp=sharing)
-- [CalculatorTest](https://drive.google.com/file/d/1KUHnC5k5e3Jv0cQeXUyLHbNPGgxB6qwT/view?usp=sharing), [Calculator](https://drive.google.com/file/d/1AoLJlPgtJFsh3-NjylTCjxfVWacroPof/view?usp=sharing)
-- [CalculatorTest*](https://drive.google.com/file/d/1uUHVlfNdkfAl9LdG9Bd_ff0XtLNIGL2L/view?usp=sharing), [Calculator*](https://drive.google.com/file/d/1tbRorYRoYLovekaHerV6yh8dGR8FFTY7/view?usp=sharing)
-- [GuessNumberTest](https://drive.google.com/file/d/19i1GZWPFCGQqXYMBg5dlWBefHRw_dtdT/view?usp=sharing), [Player](https://drive.google.com/file/d/13FrnxhgBWjSdpUCTeZ3aZ6H8SNoITtWD/view?usp=sharing), [GuessNumber](https://drive.google.com/file/d/1-3rNgJTKj-5S5wlfi7FJW2fPE2sYw8Yh/view?usp=sharing)
-- [GuessNumberTest*](https://drive.google.com/file/d/15cMQHqp8_Ljy6Cz6TyC6Vn5XVGhjeU-6/view?usp=sharing), [Player*](https://drive.google.com/file/d/1iO5-CCU2uE3x1RbjgukpyPvPn0s8y2Vz/view?usp=sharing), [GuessNumber*](https://drive.google.com/file/d/1pEEyzwzMlIT_BTenB6ESCtUN3-wkzZ9H/view?usp=sharing)
-- Выпускной: [BookshelfTest](https://drive.google.com/file/d/1z12bzUUJVVR-64CgsWUcXEGkIrQ8zA3u/view?usp=sharing), [Book](https://drive.google.com/file/d/1qIIZgCG0GL0QTxBPGmv4cIjuikofLpX3/view?usp=sharing), [Bookshelf](https://drive.google.com/file/d/1k2JdlQjcSMozEvCKNrXMfbkweL1ZLJ7b/view?usp=sharing)
-
-## Темы пятого занятия
-1. [Введение в реляционные базы данных](#1)
-1. [Реляционная СУБД PostgreSQL. Терминал psql](#2)
-1. [Введение в SQL](#3)
-1. [Домашнее задание](#4)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="1">1. [Введение в реляционные базы данных](https://drive.google.com/file/d/16JpQcAwKyhFQkh0D6et2RMVXx2fUFBkc/view?usp=sharing)</a>
-
-**Материалы:**
-- [Что такое базы данных, СУБД и язык SQL](https://youtu.be/GbogxIMRy-o) (youtube)
-- [Основные понятия базы данных](https://youtu.be/pHjGiwhitwQ) (youtube)
-- [База данных SQL](https://youtu.be/kUFDvZVETKM) (youtube)
-
-![database2](https://user-images.githubusercontent.com/29703461/40881632-64b2d3a4-66d4-11e8-842e-366d29c783f2.png)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="2">2. [Реляционная СУБД PostgreSQL. Терминал psql](https://drive.google.com/file/d/1GQulzZEjzfvbOVcK7rXagprbpkMm0rg5/view?usp=sharing)</a>
-Скачайте и установите [СУБД PostgreSQL](https://www.postgresql.org/download)
-
-- `psql -U postgres` — запуск `psql` в Windows
-- `sudo -u postgres psql` — запуск `psql` в Linux
-
-> - Для запуска `psql` из консоли необходимо [занести](https://bestprogrammer.ru/baza-dannyh/podklyuchitsya-k-komandnoj-stroke-bazy-dannyh-postgresql-windows) в переменную `PATH` путь до папки `bin`
-> - При подключении к базе данных потребуется ввести пороль, ввод которого [не будет отображаться](https://stackru.com/questions/847564/psql-nevozmozhno-vvesti-parol)
-
-**Материалы:**
-- [Установка и настройка PostgreSQL в Windows](https://youtu.be/aLDMDR8FKuk) (youtube)
-- [Установка и настройка PostgreSQL в Ubuntu](https://youtu.be/n4sNHdnXj6Q) (youtube)
-- [Документация к PostgreSQL](https://postgrespro.ru/docs/postgresql/15/index)
-- [Документация к psql](https://postgrespro.ru/docs/postgresql/15/app-psql) 
-- [DB-Engines Ranking](https://db-engines.com/en/ranking)
-- [PostgreSQL GUI Tools](https://wiki.postgresql.org/wiki/Community_Guide_to_PostgreSQL_GUI_Tools)
-- [Книги по PostgreSQL](https://postgrespro.ru/education/books)
-
-![postgresql-10](https://user-images.githubusercontent.com/29703461/40881654-c0325af6-66d4-11e8-9a40-b7de3fb24f7b.png)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="3">3. [Введение в SQL](https://drive.google.com/file/d/1NTxcqFXVFgEHcQGNhX8GkchXgdvFs-g3/view?usp=sharing)</a>
-
-**Материалы:**
-- [SQL ключи во всех подробностях](https://habr.com/company/oleg-bunin/blog/348172/)
-- [Обучалка SQL](http://www.sql-ex.ru/)
-- [SQL для простых смертных](https://ozon.ru/t/9oR10jk)
-- [Интерактивный тренажер по SQL](https://stepik.org/course/63054/promo)
-- [Интерактивная обучалка по PostgreSQL](https://www.pgexercises.com/)
-- [Руководство по стилю SQL](https://www.sqlstyle.guide/ru/)
-
-## <a name="4">4. Домашнее задание</a>
-- [Домашнее задание на тему “Базы данных”](https://docs.google.com/document/d/1f7JXmsZ0ZF7ECw814GzgG7BcpEDZvhbO8vBbWbiab3w/edit?usp=sharing)
-- [Задание на подсчет строк кода в проекте](https://docs.google.com/document/d/1rojIPnERaCn0_0VVM_YvBHEiwrB7NwzGyHHJDnq5v0Y/edit?usp=sharing)
-
-<details>
-<summary>Так должна выглядеть структура ваших папок и файлов</summary>
-
-![tree5](https://github.com/ichimax/startjava/assets/29703461/08d0ef60-0c64-440e-ab80-8eb3e19aa3dc)
-</details>
Index: ../startjava-master/README.md
===================================================================
diff --git a/../startjava-master/README.md b/../startjava-master/README.md
deleted file mode 100644
--- a/../startjava-master/README.md	
+++ /dev/null	
@@ -1,120 +0,0 @@
-# StartJava — курс программирования на Java для начинающих
-
-![header](https://user-images.githubusercontent.com/29703461/177752588-4b3a45fc-3abf-4d9f-ada4-fa3cbc25f315.png)
-
-Если вы оказались на этой странице, значит, решили начать изучать язык программирования Java. Если вы достаточно упорны и терпеливы, то рано или поздно старания принесут ожидаемые плоды — вы освоите профессию разработчика. В современном мире — это один из немногих видов деятельности, который позволит вам чувствовать себя прекрасно во всех отношениях. Но достичь этого положения очень не просто: не каждому суждено в итоге стать программистом.
-
-Чтобы облегчить ваше обучение, помочь начать программировать, видеть прогресс и повысить шансы на успех, мы разработали авторский курс StartJava, который значительно ускорит процесс освоения Java, заложив прочный фундамент для дальнейшего и более глубокого его изучения. Курс StartJava является первым шагом на пути к профессии Java-программист. Именно с него следует начинать изучать этот язык.
-
-**Отметим достоинства курса:**
-- тщательная и качественная проверка ДЗ, минимум, два раза в день
-- ответы наставником на любые вопросы, связанные с изучаемыми темами
-- ДЗ проверяется до тех пор, пока оно не будет решено
-- терпеливый наставник без вредных привычек :)
-- большое количество практических заданий разной сложности на каждую тему
-- множество тестов на пройденные темы после каждого урока
-- написание выпускного проекта и ревью его кода наставником
-- авторские видеолекции и статьи с пошаговым разбором материала
-- обилие дополнительных источников для углубленного изучения материала
-- учебный материал хорошо структурирован, последователен и понятен: подается с постепенно возрастающей сложностью
-- обучение не только Java, но и всем сопутствующим технологиям и инструментам: от работы в консоли до Git и баз данных
-- обучение техникам Clean Code, Refactoring и форматированию кода с первого урока
-- получение навыка писать красивый и читаемый код — его качество вырастет в разы
-- привитие культуры профессиональной работы с кодом и правильных привычек
-- закладывание качественного и серьезного фундамента для дальнейшего изучения Java
-- развитие мышления программиста, а также умения искать решение проблем
-- подготовитесь к поступлению на курс BaseJava
-- наконец-то почувствуете прогресс от обучения и начнете программировать
-
-**Общие сведения:**
-- электронный сертификат по окончанию на русском и английском языках
-- среднее время обучения 5 - 7 недель
-- курс можно проходить в удобные для себя время и темп
-- старт обучения сразу после оплаты
-- по завершению - скидка на курс BaseJava
-
-**Формат обучения:**
-- вы смотрите наши авторские видеолекции, изучаете дополнительный материал, выполняете ДЗ
-- по мере готовности сообщаете наставнику, что его нужно проверить
-- он проверяет и пишет, что требуется исправить
-- вы исправляете, он снова проверяет. И так повторяется до тех пор, пока ваше решение не будет соответствовать условию ДЗ и представлению наставника о том, как его нужно выполнить
-- если все верно, то наставник напишет, чтобы вы выполняли следующее ДЗ
-
-Курс очень интересный и насыщенный, постоянно развивается и улучшается. Если еще есть сомнения, то [ознакомьтесь с отзывами](https://vk.com/topic-18505771_38447031?offset=80) выпускников.
-
-Также предлагаем вам посмотреть наш вебинар на тему ["Как стать Java-программистом"](https://vk.com/wall-18505771_829).
-
-## Программа интенсива
-- **Занятие первое**
-  - [Подготовка рабочего окружения](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-1-подготовка-рабочего-окружения)
-  - [Первая программа](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-2-первая-программа)
-  - [Ручная компиляция и запуск программы](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-3-Ручная-компиляция-и-запуск-программы)
-  - [Этапы компиляции и запуска](https://github.com/ichimax/startjava/blob/master/lesson%201.md#4-этапы-компиляции-и-запуска)
-  - [Байт-код](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-5-байт-код)
-  - [Переменные: примитивные типы данных](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-6-Переменные-примитивные-типы-данных)
-  - [Ветвление: if, if else](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-7-Ветвление-if-if-else)
-  - [Циклы: for, while, do while](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-8-Циклы-for-while-do-while)
-  - [Система управления версиями Git](https://github.com/ichimax/startjava/blob/master/lesson%201.md#-9-Система-управления-версиями-git)
-  - [Итоговые домашние задания](https://github.com/ichimax/startjava/blob/master/lesson%201.md#10)
-  - [Опросники](https://github.com/ichimax/startjava/blob/master/lesson%201.md#11)
-  - [Общие рекомендации по выполнению ДЗ](https://github.com/ichimax/startjava/blob/master/lesson%201.md#12)
-  
-- **Занятие второе**
-  - [Разбор домашнего задания](https://github.com/ichimax/startjava/blob/master/lesson%202.md#Разбор-домашнего-задания)
-  - [Что такое ООП](https://github.com/ichimax/startjava/blob/master/lesson%202.md#-1-Что-такое-ООП)
-  - [Что такое класс](https://github.com/ichimax/startjava/blob/master/lesson%202.md#-2-Что-такое-класс)
-  - [Что такое объект](https://github.com/ichimax/startjava/blob/master/lesson%202.md#-3-Что-такое-объект)
-  - [Инкапсуляция: геттеры/сеттеры](https://github.com/ichimax/startjava/blob/master/lesson%202.md#-4-Инкапсуляция-геттерысеттеры)
-  - [Итоговые домашние задания](https://github.com/ichimax/startjava/blob/master/lesson%202.md#6)
-  - [Опросники](https://github.com/ichimax/startjava/blob/master/lesson%202.md#7)
-  
-- **Занятие третье**
-  - [Разбор домашнего задания](https://github.com/ichimax/startjava/blob/master/lesson%203.md#Разбор-домашнего-задания)
-  - [Механизм пакетов (package) в Java](https://github.com/ichimax/startjava/blob/master/lesson%203.md#-1-Механизм-пакетов-package-в-java)
-  - [Компиляция Java-программ. Часть II](https://github.com/ichimax/startjava/blob/master/lesson%203.md#-2-Компиляция-java-программ-Часть-ii)
-  - [.gitignore](https://github.com/ichimax/startjava/blob/master/lesson%203.md#-3-gitignore)
-  - [Домашнее задание](https://github.com/ichimax/startjava/blob/master/lesson%203.md#5-Домашнее-задание)
-  - [Опросники](https://github.com/ichimax/startjava/blob/master/lesson%203.md#6)
-  
-- **Занятие четвертое**
-  - [Разбор домашнего задания](https://github.com/ichimax/startjava/blob/master/lesson%204.md#Разбор-домашнего-задания)
-  - [Среда разработки — IntelliJ IDEA](https://github.com/ichimax/startjava/blob/master/lesson%204.md#1)
-  - [Массивы. Цикл for each](https://github.com/ichimax/startjava/blob/master/lesson%204.md#2)
-  - [Задачки на массивы](https://github.com/ichimax/startjava/blob/master/lesson%204.md#3)
-  - [Сортировка массива методом пузырька](https://github.com/ichimax/startjava/blob/master/lesson%204.md#4)
-  - [Класс java.util.Arrays](https://github.com/ichimax/startjava/blob/master/lesson%204.md#5)
-  - [Итоговые домашние задания](https://github.com/ichimax/startjava/blob/master/lesson%204.md#6)
-  - [Выпускной проект](https://github.com/ichimax/startjava/blob/master/lesson%204.md#7)
-  - [Опросники](https://github.com/ichimax/startjava/blob/master/lesson%204.md#8)
-  - [Бонусные видео](https://github.com/ichimax/startjava/blob/master/lesson%204.md#9)
-  
-- **Занятие пятое (бонусное)**
-   - [Введение в реляционные базы данных](https://github.com/ichimax/startjava/blob/master/lesson%205.md#-1-Введение-в-реляционные-базы-данных)
-   - [Реляционная СУБД PostgreSQL. Терминал psql](https://github.com/ichimax/startjava/blob/master/lesson%205.md#-2-Реляционная-СУБД-postgresql-Терминал-psql)
-   - [Введение в SQL](https://github.com/ichimax/startjava/blob/master/lesson%205.md#-3-Введение-в-sql)
-   - [Домашнее задание](https://github.com/ichimax/startjava/blob/master/lesson%205.md#4-Домашнее-задание)
-
-## Рекомендуемые книги
- - [Изучаем Java, 2-е издание](https://www.ozon.ru/context/detail/id/7821666/) (вышло [3-е издание](https://blogs.oracle.com/javamagazine/post/head-first-java-third-edition))
- - [Pro Git](https://git-scm.com/book/ru/v2)
- - [Объектно-ориентированный подход. Мэтт Вайсфельд](https://ozon.ru/t/j67EpXr)
- - [Java Programming for Kids (есть русский перевод)](http://myflex.org/books/java4kids/java4kids.htm)
- 
-## Полезные плейлисты
- - [Яков Файн: "Введение в программирование на языке Java"](https://www.youtube.com/playlist?list=PLkKunJj_bZefB1_hhS68092rbF4HFtKjW)
- - [Мирончик И.Я.: "Основы программирования на языке Java"](https://www.youtube.com/playlist?list=PL4535B9D2F8203AC7)
- 
-## Полезные ссылки
- - [Вопросы и ответы для подготовки к собеседованию](https://github.com/enhorse/java-interview)
- - [Программа для сохранения вашего зрения](https://justgetflux.com/)
- - [Топовый курс для изучения английского с нуля](https://youtube.com/playlist?list=PLD6SPjEPomauFCdDQwuHubP7F2yIVJnwN)
- - [Google Переводчик - плагин для браузера](https://chrome.google.com/webstore/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb?hl=ru)
- - [Клевая музыка для кодинга](https://www.youtube.com/channel/UCwVQIkAtyZzQSA-OY1rsGig)
- 
-## Благодарности
- - Спасибо всем, кто дает обратную связь по урокам — она помогает нам делать этот курс еще лучше
- 
- ## Контакты
- - Информация о курсах, процессе обучения, трудоустройстве: [topjava.ru](https://topjava.ru/)
- - Наша группа в [vk](https://vk.com/topjavaru)
- - Мой telegram: *ch1max*
Index: ../startjava-master/lesson 3.md
===================================================================
diff --git a/../startjava-master/lesson 3.md b/../startjava-master/lesson 3.md
deleted file mode 100644
--- a/../startjava-master/lesson 3.md	
+++ /dev/null	
@@ -1,55 +0,0 @@
-# Занятие третье
-
-## Разбор домашнего задания
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Jaeger (setters)](https://drive.google.com/file/d/1_ljcNYoajX_N48bYCRBAQOCR18telTUp/view?usp=sharing)
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Jaeger (constructors)](https://drive.google.com/file/d/1j-7-cA_WmlCQC_-2MgAMuQTUpKKE4ylt/view?usp=sharing)
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Калькулятор](https://drive.google.com/file/d/1v1MaEFTnHSpYsy4pF0a1D6kiJqQaf2he/view?usp=sharing)
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Угадай число](https://drive.google.com/file/d/1AFfvUiPiR87noEFrX8n7y9Tet57oHvEq/view?usp=sharing)
-
-**Эталонный код ДЗ 2-го урока:**
-  - [Person](https://drive.google.com/file/d/1kmYS5yoE1oFkl6llfdXSeN6vBgPFlomn/view?usp=sharing)
-  - [Wolf](https://drive.google.com/file/d/13zAxbQCWZr1HEkmPoUUqe04I9YVHZyfj/view?usp=sharing), [WolfTest](https://drive.google.com/file/d/1owoelQMWWmKoXKsqZW7u5T5NnzFMBXnZ/view?usp=sharing)
-  - [Wolf*](https://drive.google.com/file/d/1OA8zRmdA2T7YNISmTORoLM_ceAjNZN3Z/view?usp=sharing), [WolfTest*](https://drive.google.com/file/d/1RLaq3GXke5T2OaP7YB61CzvegfDAAKxw/view?usp=sharing)
-
-## Темы третьего занятия
-1. [Механизм пакетов (package) в Java](#1)
-1. [Компиляция Java-программ. Часть II](#2)
-1. [.gitignore](#3)
-1. [Домашнее задание](#5)
-1. [Опросники](#6)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 1. <a name="1">[Механизм пакетов (package) в Java](https://drive.google.com/file/d/1dzZwKVirUys88V5_CVM0RfQ4iQcQ0cIq/view?usp=sharing)</a>
-![package](https://github.com/ichimax/startjava/assets/29703461/bb23ca5c-947e-47d7-b2d2-f5d8838fdf37)
-
-**Материалы:**
-- [Руководство по пакетам в Java](https://topjava.ru/blog/rukovodstvo-po-paketam-v-java)
--	[Пакеты](https://www.youtube.com/watch?v=a6KGNASOtK8) (youtube)
-- [Packages](https://dev.java/learn/packages/)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 2. <a name="2">[Компиляция Java-программ. Часть II](https://drive.google.com/file/d/13re6jwLbagQaIkmBPr3LNUc3hIEGUbiZ/view?usp=sharing)</a>
-**Материалы:**
-- [Компиляция и запуск](https://topjava.ru/blog/rukovodstvo-po-paketam-v-java#6)
-- [Работа с Java в командной строке](https://habr.com/post/125210/)
-- [javac — документация по Java-компилятору](https://docs.oracle.com/en/java/javase/19/docs/specs/man/javac.html)
-- [java — документация по java-лаунчеру](https://docs.oracle.com/en/java/javase/19/docs/specs/man/java.html)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 3. <a name="3">[.gitignore](https://drive.google.com/file/d/1cJVmgrIfLo4iNDhfNkrdkkSZUKszzApN/view?usp=sharing)</a>
-**Материалы:**
-- [Игнорирование файлов и папок](https://topjava.ru/blog/vvedeniye-v-git-github-bazovyye-komandy#11)
-- [Игнорирование файлов в Git](https://youtu.be/fzmCx6FLLu0) (youtube)
-- [Git — игнорирование файлов](https://www.youtube.com/watch?v=EjRQ8qccLCQ) (youtube)
-- [Как создать файл .gitignore в Windows](https://ru.stackoverflow.com/questions/438367/Как-создать-файл-gitignore-в-windows/438370)
-- [Что должно и не должно быть в .gitignore](https://ru.stackoverflow.com/questions/474556/Что-должно-и-не-должно-быть-в-gitignore-для-любого-языка-и-ide)
-
-## 4. <a name="5">Домашнее задание</a>
-- **[Домашние задания по теме "Пакеты"](https://docs.google.com/document/d/1UGPow_Tqdq3GjOvBBkEsiWUu5R1NMcE83tW-Cnosu8E/edit?usp=sharing)**
-- **[Домашнее задание на форматирование кода](https://docs.google.com/document/d/19IpFE5YFkGXVdnINh1xkdQC4bWj4dfEdJwrGguFQNtQ/edit?usp=sharing)**
-<details> 
-<summary>Так должна выглядеть структура ваших папок и файлов (на GitHub папку out не загружайте)</summary>
-
-![tree2](https://github.com/ichimax/startjava/assets/29703461/123ff7f5-62be-4ff5-8659-8fc70fd23dfc)
-</details>
-
-## 5. <a name="6">Опросники</a>
-- [Итоговый тест](https://forms.gle/TqeeTy7QvUyCpsaj9) позволит вам проверить, насколько хорошо вы освоили пройденные темы
-- В этом небольшом [опроснике](https://forms.gle/wrnyKTeMoDJggTNW7) вы сможете поделиться своими впечатлениями (анонимно) по поводу третьего урока: что (не)понравилось, как его можно улучшить и тд. Ваши ответы помогут нам доработать урок, сделав его еще лучше! 
Index: ../Lesson_1/HW_lesson_1/Guess.java
===================================================================
diff --git a/../Lesson_1/HW_lesson_1/Guess.java b/../Lesson_1/HW_lesson_1/Guess.java
deleted file mode 100644
--- a/../Lesson_1/HW_lesson_1/Guess.java	
+++ /dev/null	
@@ -1,26 +0,0 @@
-import java.util.Random;
-import java.util.Scanner;
-
-public class Guess{
-	
-	public static void main(String[] args) {
-		Random rand = new Random();
-		Scanner console = new Scanner(System.in);
-
-      	
-		int randomNumber = rand.nextInt(100);
-
-		while(true){
-			int userNumber = console.nextInt();
-			if (userNumber == randomNumber) {
-				System.out.print("Число угадано");
-				break;
-			}else{
-				if (randomNumber - userNumber > 0) {
-					System.out.println("Число меньше ожадаемого");
-						}else{System.out.println("Число больше ожадаемого");}
-			}
-		}
-
-	}
-}
\ No newline at end of file
Index: ../startjava-master/lesson 4.md
===================================================================
diff --git a/../startjava-master/lesson 4.md b/../startjava-master/lesson 4.md
deleted file mode 100644
--- a/../startjava-master/lesson 4.md	
+++ /dev/null	
@@ -1,115 +0,0 @@
-# Занятие четвертое
-
-## Разбор домашнего задания
-- ![video](https://user-images.githubusercontent.com/29703461/81983788-359a6c80-9634-11ea-9b47-09a56fd3d999.png) [Наводим порядок в проекте](https://drive.google.com/file/d/1IYfWYTgZnuJM80OkyWC7Ydpa8cgmfZOt/view?usp=sharing)
-
-**Эталонный код и компиляция/запуск:**
-- [options](https://drive.google.com/file/d/13ly63_-8cK83s1Ps7Eq8UdY6uprQ0PDU/view?usp=sharing), [optionsj](https://drive.google.com/file/d/1EhhYPhLoLxyGuWiy2xjsCHA7LkrM2L20/view?usp=sharing)
-- [RpsGame](https://drive.google.com/file/d/1mmB7fV8Uo4MHC4g0WOtEED-uDsMth94x/view?usp=sharing)
-
-## Темы четвертого занятия
-1. [Среда разработки IntelliJ IDEA](#1)
-1. [Массивы. Цикл foreach](#2)
-1. [Задачки на массивы](#3)
-1. [Сортировка массива методом пузырька](#4)
-1. [Класс java.util.Arrays](#5)
-1. [Итоговые домашние задания](#6)
-1. [Выпускной проект](#7)
-3. [Опросники](#8)
-4. [Бонусные видео](#9)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="1">1. [Среда разработки IntelliJ IDEA](https://drive.google.com/file/d/1EztSgbHW7rOMqFdIQU2gfV9DVn9zQ7q0/view?usp=sharing)</a>
-> Скачайте и установите [IntelliJ IDEA Community](http://www.jetbrains.com/idea/download/index.html)
-
-**Материалы:**
-- [IntelliJ IDEA для новичков: советы и секреты](https://www.youtube.com/watch?v=mcvnjaLqVWQ) (youtube)
-- [Горячие клавиши Intellij IDEA](https://devcolibri.com/горячие-клавиши-intellij-idea)
-- [Отладка Java кода в IDEA. Основные возможности отладчика](https://youtu.be/Z1BQsf0A4xY?si=E8RtUNE1Pm3xP_w9) (youtube)
-
-![idea](https://user-images.githubusercontent.com/29703461/165950777-7fbb404c-4cca-41c4-8c51-6a7700b5c838.png)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="2">2. [Массивы](https://drive.google.com/file/d/11hTa-7sbV2R7YKNmfE4qZ_4DBtIwUKwJ/view?usp=sharing). Цикл [foreach](https://drive.google.com/open?id=1ZvCrFOKiIV01ZO1yGtDsuFRNg3YksfqU)</a>
-> Массив — это структура данных фиксированного размера, являющаяся объектом и состоящая из ячеек, расположенных последовательно в памяти, которые могут хранить в себе значения только одного, заранее заданного типа. При этом каждая ячейка обладает адресом (порядковым номером, индексом), позволяющим получать к ней доступ.
-
-**Материалы:**
-- [Руководство по массивам в Java](https://topjava.ru/blog/rukovodstvo-po-massivam-v-java-ch1)
-- [Массивы](http://developer.alexanderklimov.ru/android/java/array.php)
-- Массивы в Java ([1](http://www.skipy.ru/technics/arrays.html), [2](https://vertex-academy.com/tutorials/ru/massivy-v-java/))
-- [Varargs](https://topjava.ru/blog/rukovodstvo-po-massivam-v-java-varargs)
-- [Arrays](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html)
-- [Массивы в Java](https://www.youtube.com/watch?v=li86TEAEhYM) (youtube)
-- [Цикл foreach, Массивы строк](https://www.youtube.com/watch?v=8AD55r64yNw) (youtube)
-- [Цикл foreach в Java](https://pr0java.blogspot.com/2015/04/for-foreach.html)
-
-![arr](https://user-images.githubusercontent.com/29703461/40573705-dd7d8a52-60cd-11e8-8213-7f79b0c56f24.png)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a name="3">3. [Задачки на массивы](https://drive.google.com/file/d/1Bp7evzvvumsISUp0DHXBekH0iDOflkvB/view?usp=sharing)</a>
-
-- **[Домашние задания по теме "Массивы. Цикл foreach"](https://docs.google.com/document/d/1pDPnaFHxf5QDrO7_lCjncmyGuAfjV6bhmhhn-eh7fMg/edit?usp=sharing)**
-
-**Материалы:**
- - [Задачи на массивы в Java](http://taskcode.ru/array)
- - [Java Array: Exercises, Practice, Solution](https://www.w3resource.com/java-exercises/array/index.php)
- - Array [1](http://codingbat.com/java/Array-1), [2](http://codingbat.com/java/Array-2), [3](http://codingbat.com/java/Array-3)
-
-Ссылки на задачи из раздела *Материалы* даны для самостоятельного изучения. Ментором они не проверяются. При этом все задачи решать не нужно. Решите то количество, которое позволит вам разобраться в следующих вопросах о массиве:
-- как его создать
-- как обращаться к его ячейкам по индексу
-- как в него помещать значения и брать их из него
-- как работать с ним в цикле (заполнять, выводить на консоль, делать проверки)
- 
-## <a name="4">4. Сортировка массива методом пузырька</a>
-
-**Материалы:**
-- [Сортировка пузырьком](https://ru.wikipedia.org/wiki/Сортировка_пузырьком) (wiki)
-- [Cортировка пузырьком (bubble sort)](https://www.youtube.com/watch?v=oqpICiM165I) (youtube)
-- [Алгоритм сортировка методом пузырька](https://www.youtube.com/watch?v=5JMInXAtnQg) (youtube)
-- [Сортировка пузырьком в Java](https://nicholasgribanov.name/legendarnaya-sortirovka-puzyrkom/)
-- [Сортировки в гифках: 8 самых популярных алгоритмов](https://proglib.io/p/sort-gif)
-
-![sort](https://user-images.githubusercontent.com/29703461/40580205-cc8a3554-6142-11e8-9776-e3e20817fd5b.gif)
-
-## <a name="5">5. Класс java.util.Arrays</a>
-
-**Материалы:**
-- [Классы Arrays и System](https://topjava.ru/blog/rukovodstvo-po-massivam-v-java-ch3)
-- [Класс Arrays](http://developer.alexanderklimov.ru/android/java/array.php#arrays)
-- [Class Arrays API](https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/util/Arrays.html)
-- [Бинарный поиск](https://youtu.be/SW_UCzFO7X0?t=16m44s) (youtube)
-- [Бинарное дерево поиска](https://youtu.be/HBMlhZAOhoI) (youtube)
-
-## <a name="6">6. Итоговые домашние задания</a>
-
-- Откройте в IntelliJ IDEA папку `StartJava` со всеми домашними заданиями из прошлых уроков:
-
-![newproject](https://github.com/ichimax/startjava/assets/29703461/550d60ce-6a91-4427-ba87-751ba98ce4ce)
-
-- Прочитайте про массивы в книге [Изучаем Java](https://www.ozon.ru/context/detail/id/7821666/): стр 89, 113
-- **[Итоговые домашние задания](https://docs.google.com/document/d/1kyGSmbDsisgg94xpfcRjUKKY8DQErE0_tiZSP8-buIU/edit?usp=sharing)**
-<details>
-<summary>Так должна выглядеть структура ваших папок и файлов (на GitHub папку out не загружайте)</summary>
- 
-![tree4](https://github.com/ichimax/startjava/assets/29703461/08566887-d292-44c2-a9dd-f5d405f4b9ec)
-</details>
-
-## 7. <a name="7">Выпускной проект</a>
-- Вы можете выполнить [выпускной проект](https://docs.google.com/document/d/1IAiDJmSAD86P-xTp3kOs-tFFuDQViXRewI-vpk1nT3g/edit?usp=sharing), который проверит наставник. Это не обязательная опция, делайте его по желанию
-
-## 8. <a name="8">Опросники</a>
-- [Итоговый тест](https://forms.gle/gqgWh3wh1WbWKBWx7) позволит вам проверить, насколько хорошо вы освоили пройденные темы
-- В этом небольшом [опроснике](https://forms.gle/ykP2B4KsbLzMxEh6A) вы сможете поделиться своими впечатлениями (анонимно) по поводу четвертого урока: что (не)понравилось, как его можно улучшить и тд. Ваши ответы помогут нам доработать урок, сделав его еще лучше!
-  
-## 9. <a name="9">Бонусные видео</a>
-- [Видео](https://drive.google.com/file/d/1W9AtOE3yvDCOOZxmHSw4lGpFiFVYJB8_/view?usp=sharing) с разбором того, как необходимо решить вступительное задание к курсу basejava
-- Также предлагаем вам посмотреть наш вебинар на тему ["Как стать Java-программистом"](https://vk.com/wall-18505771_829)
-
-## Полезные сочетания клавиш в IntelliJ IDEA
-- `Ctrl + Space` — автодополнение кода
-- `Alt + Shift + F10`	— выбрать файл запуска и выполнить программу
-- `Ctrl + k` — выполнить коммит
-- `Ctrl + Shift + k` — выполнить пуш
-- `Ctrl + p` — сведения о параметрах вызываемого метода
-- `Ctrl + n` — найти класс и перейти к нему
-- `Ctrl + F12` — список всех полей и методов класса
-- `Ctrl + /` — однострочное комментирование / раскомментирование
-- `Ctrl + q` — отобразить справку к методу
Index: HW_2/WolfTest.java
===================================================================
diff --git a/HW_2/WolfTest.java b/HW_2/WolfTest.java
deleted file mode 100644
--- a/HW_2/WolfTest.java	
+++ /dev/null	
@@ -1,21 +0,0 @@
-public class WolfTest{
-
-	public static void main(String[] args) {
-		Wolf first = new Wolf();
-		first.go(7);
-		first.set_doun();
-		first.sprint(70);
-		System.out.println("Вес в начале " + first.getWeigt());
-		first.setWeigt(40);
-		System.out.println("Вес в конце " + first.getWeigt());
-		System.out.println("Имя пса " + first.getName());
-
-		Wolf second = new Wolf();
-		System.out.println("Вес в начале " + second.getWeigt());
-		second.setWeigt(10);
-		System.out.println("Вес в конце " + second.getWeigt());
-
-		System.out.println("Вес первого пса " + first.getWeigt());
-		System.out.println("Вес второго пса " + second.getWeigt());
-	}
-}
\ No newline at end of file
Index: ../startjava-master/lesson 1.md
===================================================================
diff --git a/../startjava-master/lesson 1.md b/../startjava-master/lesson 1.md
deleted file mode 100644
--- a/../startjava-master/lesson 1.md	
+++ /dev/null	
@@ -1,188 +0,0 @@
-> ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) — эта иконка означает, что рядом с ней находится ссылка, содержащая видеоурок: кликайте на нее для его просмотра
-
-## [Отзывы выпускников о курсе](https://vk.com/topic-18505771_38447031?offset=100)
-## [Демонстрация проверки ДЗ наставником](https://youtu.be/H_PYOW9Qq80?si=isPC3-aYYPhDWL5B)
-
-# ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) <a>[Занятие первое](https://drive.google.com/file/d/1l2g_mGPw3XEJ9OyURaXosRVqmeDz4kId/view?usp=sharing)</a>
-
-## Темы занятия
-1. [Подготовка рабочего окружения](#1)
-1. [Первая программа](#2)
-1. [Ручная компиляция и запуск программы](#3)
-1. [Этапы компиляции и запуска](#4)
-1. [Байт-код](#5)
-1. [Переменные: примитивные типы данных](#6)
-1. [Ветвление: if, if else](#7)
-1. [Циклы: for, while, do while](#8)
-1. [Система управления версиями Git](#9)
-1. [Итоговые домашние задания](#10)
-1. [Опросники](#11)
-1. [Общие рекомендации по выполнению ДЗ](#12)
-
-# [Видеообзор курса StartJava](https://youtu.be/vM1ygW7pKTE)
-- На данный момент обучение проводится в Slack в индивидуальном режиме
-- Обязательно читайте книги из [списка](https://topjava.ru/blog/spisok-knig-dlya-nachinayushchego-java-programmista) параллельно изучаемым темам
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 1. <a name="1">[Подготовка рабочего окружения](https://drive.google.com/file/d/1gWrBFrly3BdXKq868Xr0QkGtpM9FCmI4/view?usp=sharing)</a>
-
-- [Установите и настройте Java](https://topjava.ru/blog/ustanovka-i-nastroyka-java)
-- Скачайте и установите [Sublime Text 4](https://www.sublimetext.com/download). Именно в нем мы будем писать наши программы (с четвертого урока перейдем на IntelliJ IDEA)
-
-**[Отключите](https://chromefaq.ru/nastrojki/kak-perevodit-stranitsy-v-google-chrome-s-anglijskogo-na-russkij#i-6) в браузере автоперевод страниц. Из-за него уроки могут отображаться некорректно**
-
-**Материалы:**
-- [Знакомство с языком Java](https://www.youtube.com/live/QGHGR0xEcFA?si=xW98gPEVGZX_UrG5)
-- [Что такое JDK? Введение в средства разработки Java](https://topjava.ru/blog/what-is-the-jdk)
-- [Что такое JRE? Введение в среду выполнения Java](https://topjava.ru/blog/what-is-the-jre)
-- [Что такое JVM? Знакомство с виртуальной машиной Java](https://topjava.ru/blog/what-is-the-jvm)
-- Отчет о популярности Java-технологий в [2019](https://topjava.ru/blog/sostoyanie-java-v-2019-godu), в 2023 ([1](https://drive.google.com/file/d/1KAqubG1ETRfRioEcM46M-OKOh2ZNpzJQ/view?usp=sharing), [2](https://drive.google.com/file/d/14V5UBnbGI7pSa2_kIir36eilF36LiZOD/view?usp=sharing), [3](https://www.jetbrains.com/ru-ru/lp/devecosystem-2022/java/))
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 2. <a name="2">[Первая программа](https://drive.google.com/file/d/1anXQBZ80wvfzozUNlfhy1wtS-eS50cZi/view?usp=sharing)</a>
-
-- Изучите статью [Java с нуля: первая программа](https://topjava.ru/blog/pervaya-programma-na-java). В ней показано создание простой программы MyFirstApp с main-методом, а также дается подробный разбор каждой строки кода:
-
-``` java
-public class MyFirstApp {	
-    public static void main(String[] args) {
-        System.out.println("Write once, run anywhere");
-    }
-}
-```
-
-- [Настройте](https://topjava.ru/blog/nastroyka-sublime-text-dlya-java) Sublime Text для Java
-- [Настройте](https://topjava.ru/blog/nastroyka-sublime-text-dlya-java#7) проверку правописания для Sublime Text
-
-**Материалы:**
-- [Обзор и настройка Sublime Text](https://youtu.be/xWhTf_o86Lg) — видео дано для ознакомления с возможностями редактора. Это не значит, что вам нужно делать все то, что делает его автор (youtube)
-- [Установка плагина Terminal для SublimeText](https://youtu.be/9cFW481D2lU) (youtube) — можно использовать либо этот плагин, либо `Terminus`
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 3. <a name="3">[Ручная компиляция и запуск программы](https://drive.google.com/file/d/1u0jyGJkFFLUwfj0c21uY2i2louNtIm_3/view?usp=sharing)</a>
-Для компиляции и запуска программы необходимо в консоли перейти в папку, в которой находится файл `MyFirstApp.java`, и ввести последовательно следующие команды:
-- `javac MyFirstApp.java` — компиляция java-файла
-- `java MyFirstApp` — запуск программы
-
-Для однофайловых программ, начиная с Java 11, процесс компиляции и запуска можно совместить в одной команде:
-- `java MyFirstApp.java`
-
-Для того, чтобы русский текст корректно отображался в консоли, компилируйте свои программы с помощью специального параметра `javac -encoding utf8 имя_класса.java`
-
-Более подробно об этом рассказывается в статье [Компиляция и запуск Java-программ](https://topjava.ru/blog/kompilyatsiya-i-zapusk-programm-na-java)
-
-А из статьи ["Основы командной строки для Java-программиста"](https://topjava.ru/blog/osnovy-komandnoy-stroki-dlya-java-programmista) вы узнаете о самых нужных и часто используемых командах консоли для работы с файловой системой 
-
-**Материалы:**
-- [Getting Started with Java](https://dev.java/learn/getting-started/)
-- [Введение в командную строку Windows](https://www.youtube.com/user/Iidsp/videos) (youtube)
-
-## 4. <a name="4">Этапы компиляции и запуска</a>
-![Frame 303](https://github.com/ichimax/startjava/assets/29703461/613d6100-8fc1-4054-9811-cf143b61d32c)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 5. <a name="5">[Байт-код](https://drive.google.com/file/d/19Tl7dEpPszjQfulzQBEwyn9sQL0DK3ji/view?usp=sharing)</a>
-Для отображения байт-кода класса необходимо в консоли перейти в папку, в которой находится файл `MyFirstApp.class`, и написать:
-`javap -c -s -verbose MyFirstApp`
-
-**Материалы:**
-- [Java байт-код «Hello world»](https://habr.com/post/264919/)
-- [javap — Disassembles one or more class files](https://docs.oracle.com/en/java/javase/17/docs/specs/man/javap.html)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 6. <a name="6">[Переменные: примитивные типы данных](https://drive.google.com/file/d/1mrsKC-wU7NpxQiNt47rgfySsDGTwB8Wb/view?usp=sharing)</a>
-
-**Материалы:**
-- [Переменные в Java](https://youtu.be/Y__Ns7FS5lA?t=17) (youtube)
-- [Примитивные типы данных](http://developer.alexanderklimov.ru/android/java/types.php)
-- [Variables](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html)
-- [Primitive Data Types](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html)
-- [Про размеры типов данных](https://youtu.be/3BmznLJAgaA?t=782)
-
-![var](https://user-images.githubusercontent.com/29703461/155842985-5fae137b-3967-4fe6-893c-0d2a018815a5.png)
-
-- **Домашние задания по теме "Переменные"**
-  - ["Именование переменных"](https://docs.google.com/document/d/1Dv-eo8acvOVFJtUVbaWd1Mfb77Q1IiTlhepgy56qMbc/edit?usp=sharing)
-  - ["Примитивные типы данных"](https://docs.google.com/document/d/1kJqeCC-fKWUjBjrmHJ1mjDu45SGRL39Q5vW4DmLJflA/edit?usp=sharing)
-- Разберитесь с [Git/GitHub](https://github.com/ichimax/startjava/blob/master/lesson%201.md#9): почитайте рекомендации о том, как с ними работать, настройте свой локальный репозиторий
-- **Покажите ДЗ наставнику** (пришлите ему ссылку на ваш репозиторий с решенным ДЗ)
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 7. <a name="7">[Ветвление: if, if else](https://drive.google.com/file/d/1r-UVn4Z9OhV-SUvJdMrC6EG_v9qowAJS/view?usp=sharing)</a>
-
-**Материалы:**
-- [Ветвление в Java](https://javarush.ru/groups/posts/2726-vetvlenie-v-java)
-- [Оператор if else](https://youtu.be/ryR033ld_N0) (youtube)
-- [The if-then and if-then-else Statements](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/if.html)
-
-![turpal](https://user-images.githubusercontent.com/29703461/39215173-9afa64b4-481f-11e8-9731-fa60a8439f71.jpg)
-
-- **[Домашние задания по теме "Ветвление: if-else"](https://docs.google.com/document/d/1LZLCupFXrCmTuaT4KCUW9nKjrzaFDCCH7i5PDpWSLjI/edit?usp=sharing)**
-- **Покажите ДЗ наставнику**
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 8. <a name="8">Циклы: [for](https://drive.google.com/file/d/1MxaPlSwY6-Nv2zOrQGYy5lOSDjYTfnfU/view?usp=sharing), [while, do while](https://drive.google.com/file/d/1YQHY9C1QbueWYFMRBpHFTqg1wt9N_333/view?usp=sharing)</a>
-![loop](https://user-images.githubusercontent.com/29703461/39228479-100883f2-4867-11e8-9d63-5d18e455aaa2.jpeg)
-
-**Материалы:**
-- Циклы в Java [1](https://timeweb.cloud/tutorials/java/cikly-v-java-osnovy-raboty), [2](https://vertex-academy.com/tutorials/ru/cikly-v-java/)
-- [Оператор while](https://youtu.be/15PjODTSTaw) (youtube)
-- [Оператор for](https://youtu.be/UYbdAmgcNVc) (youtube)
-
-**Домашнее задание:**
-- **[Домашние задания по теме "Циклы: for, while, do while"](https://docs.google.com/document/d/1lHSSOzARTx6tfD-4MV-t41S5bnvdNw5hczkQPT05png/edit?usp=sharing)**
-- **Покажите ДЗ наставнику**
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 9. <a name="9">[Система управления версиями Git](https://drive.google.com/file/d/1IMw7DA_lodW2GueWpU1nMUmYx0pnlPGA/view?usp=sharing)</a>
-![image](https://user-images.githubusercontent.com/29703461/154850409-7a618576-1105-48a1-9e32-88a34e87693b.png)
-
-- [Установите и настройте Git/GitHub](https://topjava.ru/blog/vvedeniye-v-git-github-ustanovka-i-nastroyka)
-
-**Материалы:**
-  - [Базовый курс по обучению Git](https://www.youtube.com/playlist?list=PLIU76b8Cjem5B3sufBJ_KFTpKkMEvaTQR) (youtube)
-  - [Git. Быстрый старт](https://www.youtube.com/watch?v=4-NX17Ip-xQ&list=PLmRNNqEA7JoM77hOJkPrLOfJQGizCLR3P) (youtube)
-  - Интерактивные Git-обучалки: [1](https://githowto.com/ru), [2](http://learngitbranching.js.org)
-  - [Официальная книга про Git](https://git-scm.com/book/ru/v2)
-  
-### Настройка локального репозитория
-Более подробно об этом рассказывается в статье [Введение в Git/GitHub: базовые команды](https://topjava.ru/blog/vvedeniye-v-git-github-bazovyye-komandy).
-
-- Создайте в удобном для вас месте папку под названием `StartJava`. В ней будет храниться ваш код
-- Создайте на GitHub репозиторий с названием `startjava`
-- Напишите в консоли (при этом консоль должна быть открыта в папке `startjava`): 
-  - `git init` — создание локального репозитория
-  - `git status` — вывод состояния файлов
-  - добавьте адрес созданного на GitHub репозитория в git — `git remote add origin url_на_ваш_startjava-репозиторий.git`
-  - проверьте, добавился он или нет — `git remote -v`
-
-### Рекомендации по Git
-1. Последовательность стандартных шагов при работе с Git:
-   - `git add файл_с_кодом.java` — добавляем файлы/изменения в git (файлы с расширением `class` не добавляйте)
-   - `git commit -m "краткое описание изменений, которые вы внесли в программу"` — фиксируем (коммитим) изменения
-   - `git push -u origin master` — [устанавливаем связь](https://qna.habr.com/q/118865) между локальной и удаленной веткой master
-1. На GitHub можно отправлять не только файлы с расширением `java`, но и папки, например, `Lesson 1` (при этом они не должны быть пустыми)
-1. Если вы столкнулись с проблемами при использовании Git, то статья ["Ошибки использования Git"](https://topjava.ru/blog/vvedeniye-v-git-oshibki-ispolzovaniya-ch-9) поможет их решить
-
-## ![video](https://user-images.githubusercontent.com/29703461/81982928-d556fb00-9632-11ea-9794-ea198832d674.png) 10. <a name="10">[Итоговые домашние задания](https://drive.google.com/file/d/1Svgm_03td3Aj_MeVt-MBqSCv-PEhXD0A/view?usp=sharing)</a>
-
-- **[Итоговые домашние задания](https://docs.google.com/document/d/1m6y5qBMfwKjm0DIo3JYqQ9_QNyhroDvsRAirEyDEP7s/edit?usp=sharing)**
-
-<details>
-<summary>Так должна выглядеть структура ваших папок и файлов</summary>
-    
-![tree](https://github.com/ichimax/startjava/assets/29703461/47ca8fc6-3f00-4c24-a07d-0e2416f12eb6)
-</details>
-
-## 11. <a name="11">Опросники</a>
-- [Итоговый тест](https://forms.gle/YFc2pf6qZw6jdfXM7) позволит вам проверить, насколько хорошо вы освоили пройденные в уроке темы
-- В этом небольшом [опроснике](https://forms.gle/dsthsvh1tRhdJyhu9) вы сможете поделиться своими впечатлениями (анонимно) по поводу первого урока: что (не)понравилось, как его можно улучшить и тд. Ваши ответы помогут нам доработать урок, сделав его еще лучше!
-
-## 12. <a name="12">Общие рекомендации по выполнению ДЗ</a>
-
-1. [Правила](https://github.com/ichimax/startjava/blob/master/rules.md) проверки ДЗ наставником
-1. Ментору ссылку на репозиторий достаточно прислать один раз (для самого первого ДЗ). Для последующих ДЗ просто сообщайте, что оно готово и можно проверять
-1. При выполнении ДЗ используйте только те темы, которые изучались в текущем уроке
-1. Ввод с клавиатуры, методы класса `Math` или `Random` пока не используйте (но, если вы знаете про них и хотите использовать — дерзайте)
-
-## Немного мотивации
-- [Как приучить себя к дисциплине и осуществлять свои планы](https://youtu.be/wbItXmjm7tY?si=RceAgonMsijaz3KD)
-- [Shia LaBeouf Motivation for Coding](https://youtu.be/J1usv2Hn-pU)
-- [Мотивация для программистов](https://www.youtube.com/watch?v=NOVaNe3jx6c)
-- [Речь Стива Джобса Перед выпускниками Стенфорда](https://www.youtube.com/watch?v=WVz9E7o1mmk)
-- [Мотивационный совет от Стива Джобса](https://www.youtube.com/watch?v=Z2qSUZ1o47U)
-- [12 минут сильнейшей мотивации. Перелом сознания](https://www.youtube.com/watch?v=69MkvKrraC0)
-- [Почему мне так сложно учиться программировать?](https://youtu.be/TtT8UNK0sGg)
-- [Простой прием стать программистом быстрее](https://youtu.be/iw3E-Gdncwc)
Index: arrave/ArrayMain.java
===================================================================
diff --git a/arrave/ArrayMain.java b/arrave/ArrayMain.java
new file mode 100644
--- /dev/null	
+++ b/arrave/ArrayMain.java	
@@ -0,0 +1,61 @@
+package arrave;
+
+public class ArrayMain {
+
+    public static void main(String[] args) {
+        int[] numbers = new int[6]; // int - тип данных, []- массив, numbers - переменная с сылкой, = - присваивание, new создаем новый объект, int[6] - 6 ячеек в массиве
+        float[] numbers1 = {12.2f, 13.2f};
+        System.out.println(numbers[0]); // обращение к яцейки
+
+        System.out.println(numbers); //вывод [I@568db2f2 где [- одномерный массив, I- тип данных int, @ - адрес к ячейкам
+
+        int one = numbers.length;
+        System.out.println(one);
+
+
+
+
+
+        for(int i = 0; i < numbers1.length; i++){
+            System.out.print(numbers1[i] + " ");
+        }
+
+        for(int i = 0; i < numbers.length; i++){
+            numbers[i] = i * 10;
+        }
+
+        System.out.println();
+        for(int i = 0; i < numbers.length; i++){
+            System.out.print(numbers[i] + " ");
+        }
+        
+        String[] text = new String[3];
+        System.out.println(text);
+
+
+
+
+
+
+
+        //Если требуется в массиве использовать индекс элементов, то используется цикл for
+        for(int i = 0; i < numbers1.length; i++){
+            System.out.print(numbers1[i] + " ");
+        }
+
+        // Специальный цикл для массивов. По циклу поочередно прогоняется каждый элемет массива
+        for (float v : numbers1) {
+            System.out.print(v + " ");
+
+
+
+        }
+
+        }
+
+    }
+
+
+
+
+}
Index: .idea/vcs.xml
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/vcs.xml	
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: arrave/ArrayMaxElementMain.java
===================================================================
diff --git a/arrave/ArrayMaxElementMain.java b/arrave/ArrayMaxElementMain.java
new file mode 100644
--- /dev/null	
+++ b/arrave/ArrayMaxElementMain.java	
@@ -0,0 +1,12 @@
+package arrave;
+
+import java.util.Scanner;
+
+public class ArrayMaxElementMain {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+    }
+
+}
Index: .idea/misc.xml
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/misc.xml	
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="19" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: Lesson_2.iml
===================================================================
diff --git a/Lesson_2.iml b/Lesson_2.iml
new file mode 100644
--- /dev/null	
+++ b/Lesson_2.iml	
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/modules.xml
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	
+++ b/.idea/modules.xml	
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/Lesson_2.iml" filepath="$PROJECT_DIR$/Lesson_2.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	
+++ b/.idea/.gitignore	
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
